docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -196,6 +196,27 @@ There you can find links to upgrade notes for other versions too.
         -   - { resource: forms.yml }
             - { resource: services/*.yml }
         ```
+    - exclude loading of DataFixtures folder from services.yml
+        ```diff
+        -        exclude: '../../{Command,Controller,DependencyInjection,Form,Migrations,Resources,Twig}'
+        +        exclude: '../../{Command,Controller,DataFixtures,DependencyInjection,Form,Migrations,Resources,Twig}'
+        ```
+    - move all services based on `DataFixtures` namespace into `src/Shopsys/ShopBundle/Resources/config/services/data_fixtures.yml` configuration
```

Comment:
I don't understand what shall I do... Is is related with moving from `services.yml` to `data_fixtures.yml`?
If so, please rephrase the instruction so it is clear that I have to remove them from this file and paste them into this file